src/Illuminate/Http/Client/Factory.php

```diff
@@ -191,6 +191,20 @@ public function assertSent($callback)
         );
     }
 
+    /**
+     * Assert that a request / response pair was not recorded matching a given truth test.
+     *
+     * @param  callable  $callback
+     * @return void
+     */
+    public function assertNotSent($callback)
```

Comment:
Can we have a default callback that asserts that nothing as sent at all?

Reply:
Good idea. I will add another assertion called `assertNothingSent` so that it is similar to what we have with fake mails and notifications 👍

Reply:
Thanks done